### ISSUE TITLE
Suggestion: Make Public

### ISSUE THREAD

**Issue by rfearing** [chiedolabs/eslint-config-node#10]
It would be easier to utilize if it was public and nothing is sensitive in it. Any objections?

**Comment by chiedo** [comment]
Yeah. I think every web dev at Chiedo Labs should be hooked up to NPM so that should be a prerequisite. I wouldn't want to encourage people to not feel the need to set that up. 

But no objections outside of that.

**Comment by chiedo** [comment]
We'll discuss at our next Node core meeting
